test/app-tests/loading.js

```diff
@@ -343,6 +346,40 @@ describe('loading:', function () {
             }).done(done, done);
         });
 
+        it("registers correctly with a Riot Team Server", function() {
+            sdk.setFetch(httpBackend.fetchFn); // XXX: ought to restore this!
```

Comment:
Ah, in which case I'd suggest renaming the one exported in the sdk to be `setRtsFetchFn`. and it doesn't look like you're replacing fetchfn for the whole of the sdk ...